sway/desktop/render.c

```diff
@@ -1056,6 +1058,40 @@ void output_render(struct sway_output *output, struct timespec *when,
 		wlr_renderer_clear(renderer, (float[]){1, 1, 0, 1});
 	}
 
+	if (lock_state->state != SESSIONLOCK_STATE_UNLOCKED) {
```

Comment:
Maybe we should extract this block to a helper function, to make the `goto`s less confusing.